gettsim/visualization.py

```diff
@@ -400,19 +410,21 @@ def _create_pydot_layout(dag):
     for k, v in layout.items():
         layout[k] = (v - (max_ + min_) / 2) / ((max_ - min_) / 2).clip(1)
 
-    return layout
+    if orientation == "v":
 
+        layout_df = np.transpose(pd.DataFrame.from_dict(layout))
 
-def _compute_arrow_coordinates(start, end, scalar=0.05):
-    """Compute arrow coordinates.
+    elif orientation == "h":
```

Comment:
Can you also add a test for horizontan orientation and for `hover_source_code=True`?